Issues building docs on up-to-date Arch

Trying to build the docs, and I am hitting an issue:
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/sphinx/config.py", line 343, in eval_config_file
    exec(code, namespace)
  File "/home/don/tmp/khal/doc/source/conf.py", line 36, in <module>
    validator = validate.Validator()
NameError: name 'validate' is not defined
```

More complete info:

```
15:01:52 ⌁ [don@loki:~/tmp/khal] master 2 ± python setup.py --version
0.10.5.dev114+gcfa8f1e
15:02:32 ⌁ [don@loki:~/tmp/khal] master ± git log -1
commit cfa8f1e0977e7d2442ce25a8f30c2ee8a19b8e94 (HEAD -> master, origin/master, origin/HEAD)
Merge: 9007660 40e2d84
Author: Christian Geier <geier@lostpackets.de>
Date:   Sat Jun 25 21:34:30 2022 +0200

    Merge pull request #1030 from pimutils/fix/993

    Use new syntax for validate. Fixes #993.
15:02:41 ⌁ [don@loki:~/tmp/khal] master ± make -C doc man PYTHONPATH="${PWD}"
make: Entering directory '/home/don/tmp/khal/doc'
sphinx-build -b man -d build/doctrees   source build/man
Running Sphinx v5.0.2

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/sphinx/config.py", line 343, in eval_config_file
    exec(code, namespace)
  File "/home/don/tmp/khal/doc/source/conf.py", line 36, in <module>
    validator = validate.Validator()
NameError: name 'validate' is not defined

make: *** [Makefile:131: man] Error 2
make: Leaving directory '/home/don/tmp/khal/doc'
15:02:46 ⌁ [don@loki:~/tmp/khal] master 2 ± python --version
Python 3.10.5
15:03:14 ⌁ [don@loki:~/tmp/khal] master ±
```

This has worked in the past as I have been happily using khal for years.

Thanks!

--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -18,7 +18,7 @@ import khal
 
 try:
     # Available from configobj 5.1.0
-    import configobj.validate
+    import configobj.validate as validate
 except ModuleNotFoundError:
     import validate
 

